packages/stacked_tools/lib/src/services/process_service.dart

```diff
@@ -14,7 +14,7 @@ class ProcessService {
   /// Args:
   ///   appName (String): The name of the app that's going to be create.
   Future<void> runCreateApp({required String appName}) async {
-    await _runProcessAndLogOutput(
```

Comment:
Please change this back to a private function. None of the code should be running process commands directly. Put it behing a function similar to createApp so that we can manage all that code in one place.

Reply:
Extracted a new function format file and made this function private